api/graphite.go

```diff
@@ -101,63 +209,62 @@ func (s *Server) renderMetrics(ctx *middleware.Context, request models.GraphiteR
 	}
 
 	reqs := make([]models.Req, 0)
-	parsedTargets := make(map[string]string)
+
+	// consolidatorForPattern[<pattern>]<consolidateBy>
+	consolidatorForPattern := make(map[string]string)
```

Comment:
that is true, but this code works the same as what was here before (just some of the variable names have been changed)

I guess we need to make this map[string][]string?